### PR TITLE
add `cupy.random.weibull`

### PR DESCRIPTION
**Pull request by YoshikawaMasashi**

### CODE REVIEW

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

File: tests/cupy_tests/random_tests/test_generator.py [hunk 1]
```diff
@@ -727,6 +727,21 @@ def test_vonmises(self):
         self.generate(mu=self.mu, kappa=self.kappa, size=(3, 2))
 
 
+@testing.parameterize(
+    {'a': 0.5},
+    {'a': 1.0},
+    {'a': 3.0},
```
Comment (toslunar) [inline tests/cupy_tests/random_tests/test_generator.py]: Could you add testcases `0.0` and `inf`?

File: cupy/random/generator.py [hunk 2]
```diff
@@ -468,6 +468,17 @@ def vonmises(self, mu, kappa, size=None, dtype=float):
         self.rk_seed += numpy.prod(size)
         return y
 
+    def weibull(self, a, size=None, dtype=float):
+        """Returns an array of samples drawn from the weibull distribution.
+
+        .. seealso::
+            :func:`cupy.random.weibull` for full documentation,
+            :meth:`numpy.random.RandomState.weibull`
+        """
```
Comment (toslunar) [inline cupy/random/generator.py]: Could you raise error if `a` contains a negative value?

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
Jenkins, test this please.

**Pull request merged**